README.md

```diff
@@ -63,22 +72,25 @@ API Reference is located here: http://pytorch.org/audio/
 Conventions
 -----------
 
-Torchaudio is standardized around the following naming conventions.
-
-* waveform: a tensor of audio samples with dimensions (channel, time)
-* sample_rate: the rate of audio dimensions (samples per second)
-* specgram: a tensor of spectrogram with dimensions (channel, freq, time)
-* mel_specgram: a mel spectrogram with dimensions (channel, mel, time)
-* hop_length: the number of samples between the starts of consecutive frames
-* n_fft: the number of Fourier bins
-* n_mel, n_mfcc: the number of mel and MFCC bins
-* n_freq: the number of bins in a linear spectrogram
-* min_freq: the lowest frequency of the lowest band in a spectrogram
-* max_freq: the highest frequency of the highest band in a spectrogram
-* win_length: the length of the STFT window
-* window_fn: for functions that creates windows e.g. torch.hann_window
-
-Transforms expect the following dimensions. In particular, the input of all transforms and functions assumes channel first.
+With torchaudio being a machine learning library and built on top of PyTorch,
+torchaudio is standardized around the following naming conventions. In particular,
+tensors are assumed to have channel as the first dimension and time as the last
+dimension (when applicable). This makes it consistent with PyTorch's dimensions.
+
+* `waveform`: a tensor of audio samples with dimensions (channel, time)
```

Comment:
Do we want to provide explanations for the naming conventions, or just showing the examples is good enough?
* We use the conventions that channels are first.
* We call `n_*` variables referring to the number of `*`.
* etc

Reply:
good point. done.